demo/code-demo/scripts/step_authorize.js

```diff
@@ -0,0 +1,32 @@
+var embedWithSpecificReportDivExpanded = false;
+
+function ExpandEmbedWithSpecificReportDiv() {
```

Comment:
shouldn't this function name be somethings with "toggle" (that was it does doesn't it?)